src/main/java/org/folio/circulation/resources/RequestByInstanceIdResource.java

```diff
@@ -91,7 +91,7 @@
 
 public class RequestByInstanceIdResource extends Resource {
 
-  private static final List<RequestType> ORDERED_REQUEST_TYPES = List.of(HOLD, RECALL, PAGE);
+  private static final List<RequestType> ORDERED_REQUEST_TYPES = List.of(RECALL, HOLD, PAGE);
```

Comment:
Add two test scenarios
1 - RECALL is forbidden, HOLD is allowed - HOLD should be created
2 - RECALL is forbidden, HOLD is forbidden - HOLD should be created